docs/pages/getting-started/linux-server.mdx

```diff
@@ -68,6 +68,7 @@ Create a directory on your Linux machine called `demo-app` and run the following
 command:
 
 ```code
+$ mkdir demo-app
```

Comment:
Nit, but the sentence above this says that you should create a directory. Maybe change that sentence to:

```markdown
Run the following commands to create a directory on your Linux machine
called `demo-app` and add a simple HTML file to serve to clients:
```

Reply:
Updated comment.